src/pytom_tm/weights.py

```diff
@@ -53,17 +54,25 @@ def wavelength_ev2m(voltage: float) -> float:
     return _lambda
 
 
-def radial_reduced_grid(shape: tuple[int, int, int]) -> npt.NDArray[float]:
-    x = (np.abs(np.arange(
-        -shape[0] // 2 + shape[0] % 2,
-        shape[0] // 2 + shape[0] % 2, 1.
-    )) / (shape[0] // 2))[:, np.newaxis, np.newaxis]
-    y = (np.abs(np.arange(
-        -shape[1] // 2 + shape[1] % 2,
-        shape[1] // 2 + shape[1] % 2, 1.
-    )) / (shape[1] // 2))[:, np.newaxis]
-    z = np.arange(0, shape[2] // 2 + 1, 1.) / (shape[2] // 2)
-    return np.sqrt(x ** 2 + y ** 2 + z ** 2)
+def radial_reduced_grid(shape: Union[tuple[int, int, int], tuple[int, int]]) -> npt.NDArray[float]:
```

Comment:
this returns `None` if `shape not in [2,3]` is that intended? If not, please catch and throw and error.
Either way, please add a test for the behavior